src/browser/transport.js

```diff
@@ -100,7 +111,11 @@ function _proxyRequest(json, callback) {
   );
 }
 
-function _makeRequest(accessToken, url, method, data, callback, requestFactory) {
+function _isNumber(n) {
+  return !isNaN(parseFloat(n)) && isFinite(n);
+}
```

Comment:
This would be better placed in this section of src/utility.js: https://github.com/rollbar/rollbar.js/blob/v2.21.1/src/utility.js#L71-L144

Is there a reason we should allow coercion of strings, rather than requiring the the value to be a number?

Since `isFinite()` also checks for `NaN`, can we write this without `isNaN`?

Maybe:
```
function isFiniteNumber(n) {
  return Number.isFinite(n);
}
```
does everything we want here.

Reply:
Agree, moved source code